lib/python/temporal/core.py

```diff
@@ -650,21 +667,28 @@ def init(raise_fatal_error=False, skip_db_version_check=False):
         if dbif.fetchone()[0]:
             db_exists = True
 
-    backup_howto = "The format of your actual temporal database is not " \
-                   "supported any more.\nSolution: You need to export it by " \
-                   "restoring the GRASS GIS version used for creating this DB"\
-                   ". From there, create a backup of your temporal database "\
-                   "to avoid the loss of your temporal data.\nNotes: Use " \
-                   "t.rast.export and t.vect.export to make a backup of your" \
-                   " existing space time datasets.To safe the timestamps of" \
-                   " your existing maps and space time datasets, use " \
-                   "t.rast.list, t.vect.list and t.rast3d.list. "\
-                   "You can register the existing time stamped maps easily if"\
-                   " you export columns=id,start_time,end_time into text "\
-                   "files and use t.register to register them again in new" \
-                   " created space time datasets (t.create). After the backup"\
-                   " remove the existing temporal database, a new one will be"\
-                   " created automatically.\n"
+    backup_howto = _(
+        "Please create a backup of your temporal database to avoid losing data.\n"
+    )
+    if tgis_db_version == 2:
+        backup_howto += _("Run t.downgrade command to downgrade your temporal database.\n")
+    else:
```

Comment:
This `else` would be for the case of a very old temporal database (say version 1), right?

Reply:
Right, message for old (version 1) temporal databases is also in the `main` branch.